### PR TITLE
rename --runtime flag to --engine

### PR DESCRIPTION
**Pull request by ashadhaz**
As per stated in the issue #755 , I have renamed the runtime tag to engine.

### CODE REVIEW

**Comment by ashadhaz** [comment]
Thanks for guiding me @ivotron. I will work on these changes.

File: cli/popper/commands/cmd_run.py [hunk 1]
```diff
@@ -223,6 +223,8 @@ def run_workflow(**kwargs):
             pre_wf_runner = WorkflowRunner(pre_wf)
             pre_wf_runner.run(**kwargs)
 
+        for key, value in kwargs.items(): 
+            print ("%s == %s" %(key, value))
```
Comment (ivotron) [inline cli/popper/commands/cmd_run.py]: @ashadhaz, this is causing the tests to fail. If you remove this the tests should pass. thanks!

**Comment by ashadhaz** [comment]
hey @ivotron, I was able to make the build pass the tests. However there are some merge conflicts which I'm not able to resolve locally. Could you please look into it. I made an attempt to change the docstrings as well to reflect the changes but it is giving a merge conflict.

**Comment by ivotron** [comment]
thanks @ashadhaz ! I'm closing this in favor of #769

**Pull request closed without merging**